src/routers/SearchRouter.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+import LRU from 'lru-cache';
+import AllStopUtils from '../utils/AllStopUtils';
+import ApplicationRouter from '../appdev/ApplicationRouter';
+import RequestUtils from '../utils/RequestUtils';
+
+const BUS_STOP = 'busStop';
+const cacheOptions = {
+    max: 10000,
+    maxAge: 1000 * 60 * 60 * 24 * 5,
+};
+const cache = LRU(cacheOptions);
+const GOOGLE_PLACE = 'googlePlace';
+
+class SearchRouter extends ApplicationRouter<string> {
+    constructor() {
+        super(['POST']);
+    }
+
+    getPath(): string {
+        return '/search/';
+    }
+
+    async content(req): Promise<any> {
```

Comment:
can we make the type stronger than `Promise<any>`, and add a type for `req` as well?